<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="vlClient1">
        <item>
         <widget class="QLabel" name="lbl_Client1">
          <property name="text">
           <string>Client #A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_Client1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_Client1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlClient2">
        <item>
         <widget class="QLabel" name="lbl_Client2">
          <property name="text">
           <string>Client #B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_Client2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_Client2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlClient3">
        <item>
         <widget class="QLabel" name="lbl_Client3">
          <property name="text">
           <string>Client #C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_Client3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_Client3"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlServer">
        <item>
         <widget class="QLabel" name="lbl_Server">
          <property name="text">
           <string>Main Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_Server"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
